libs/packages/components/src/lib/popup/popup.directive.ts

```diff
@@ -10,19 +10,28 @@ export class SdsPopupDirective implements AfterViewInit {
   @Input() position: string;
   @Input() placement: string;
   sdsPopupDiv: HTMLElement;
+  descDiv: HTMLElement;
 
   constructor(private el: ElementRef, private renderer: Renderer2) {
     this.renderer.addClass(this.el.nativeElement, 'sds-popup');
     this.sdsPopupDiv = document.createElement('div');
     this.renderer.addClass(this.sdsPopupDiv, 'sds-popup__content');
   }
 
-
   ngAfterViewInit() {
-    this.renderer.appendChild(this.el.nativeElement, this.sdsPopup);
     this.renderer.addClass(this.sdsPopupDiv, this.placement);
     this.renderer.addClass(this.sdsPopupDiv, this.position);
+    if(typeof(this.sdsPopup) !== 'string'){
+    this.renderer.appendChild(this.el.nativeElement, this.sdsPopup);
     this.renderer.appendChild(this.sdsPopupDiv, this.el.nativeElement.children[0]);
     this.renderer.appendChild(this.el.nativeElement, this.sdsPopupDiv);
+    } else {
+      this.descDiv = document.createElement('div');
+      this.renderer.addClass(this.descDiv, 'tooltip');
+      this.renderer.addClass(this.descDiv, 'tooltip-left');
```

Comment:
tooltip-left class doesn't exist or do anything, not needed?

Reply:
tooltip-left class removed